src/internal.rs

```diff
@@ -373,19 +374,38 @@ pub type cdef_fn = Option<
 pub type cdef_dir_fn =
     Option<unsafe extern "C" fn(*const libc::c_void, ptrdiff_t, *mut libc::c_uint) -> libc::c_int>;
 
-pub type const_left_pixel_row = *const libc::c_void;
-pub type looprestorationfilter_fn = Option<
-    unsafe extern "C" fn(
-        *mut libc::c_void,
-        ptrdiff_t,
-        const_left_pixel_row,
-        *const libc::c_void,
-        libc::c_int,
-        libc::c_int,
-        *const LooprestorationParams,
-        LrEdgeFlags,
-    ) -> (),
->;
+pub type pixel_8bpc = uint8_t;
+pub type coef_8bpc = int16_t;
+
+pub type pixel_16bpc = uint16_t;
+pub type coef_16bpc = int32_t;
```

Comment:
Now that we have `trait BitDepth`, we can just use `BitDepth{8,16}::{Pixel,Coef}` for these.